kivymd/uix/filemanager.py

```diff
@@ -425,9 +423,9 @@ def __init__(self, **kwargs):
         toolbar_label = self.ids.toolbar.children[1].children[0]
         toolbar_label.font_style = "Subtitle1"
         if (
-            self.selector == "any"
-            or self.selector == "multi"
-            or self.selector == "folder"
+                self.selector == "any"
+                or self.selector == "multi"
+                or self.selector == "folder"
```

Comment:
This is not according to the `PEP8` standard.

Reply:
I use autoformatting in my compiler, I will change it now

Reply:
https://github.com/kivymd/KivyMD#setup-environment